content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Get Started
+---
+<a href="https://galaxy-mentor-network.netlify.app/" class="btn btn-primary" role="button" aria-pressed="true">Would you like to learn about Galaxy with a mentor? Check out the Galaxy Mentorship Network</a>
```

Comment:
I am not sure I would add this link here (I mean without context). As a newcomer, I don't think one would "jump" into a mentoring program without first knowing about the community itself and what it does.

Reply:
The mentorship program is mostly useful when you need guidance on a project or need help learning how to contribute to open source and for settling into the Galaxy ecosystem.

So I guess you need to "navigate" a bit the Galaxy ecosystem; at least you went through some of the Galaxy Tutorials; maybe you asked for help in the Galaxy help forum and then realise you need a bit more than this "punctual" help. So I would put it after the help forum (but is is definitely open to discussion!!!).

Reply:
I'd add just what you described as an introduction for it (or smth similar)

 _You went through some of the Galaxy Tutorials; maybe you asked for help in the Galaxy help forum and then realise you need a bit more than this "punctual" help. What about reaching to the Galaxy Mentorship Network?_